Scotch/Tools.cs

```diff
@@ -0,0 +1,21 @@
+using System.Runtime.CompilerServices;
```

Comment:
`Utils` may make more sense as a filename?

Reply:
Resolved in [ad78b60](https://github.com/EasyPost/scotch/pull/1/commits/ad78b60d7fb4845421e9b717c6e3bb519050be06)